apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/repository/RoleRepository.java

```diff
@@ -12,4 +15,9 @@
    * find role by role name
    */
   Role findTopByRoleName(String roleName);
+
+  @Query("select r.id from Role r where r.roleName = CONCAT('Master+', ?1) " +
+          " or r.roleName like CONCAT('ModifyNamespace+', ?1, '+%') or" +
+          " r.roleName like CONCAT('ReleaseNamespace+',?1, '+%')")
+  List<Integer> findByAppId(String appId);
```

Comment:
we don't need a query like this since in the program, we do know the role names, so we could build the role names using [RoleUtils](https://github.com/ctripcorp/apollo/blob/master/apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/util/RoleUtils.java) and query by role names